### ISSUE TITLE
Bug: scanner.py keeps scanning after ctrl-c

### ISSUE THREAD

**Issue by jsmif** [nccgroup/Sniffle#97]
I was sniffing connection attempts to a device in my house when I saw something was scanning it. Based on the RSSI I assumed it was coming from inside my house. I turned off all the usual culprits like android phones, but it was still being scanned. Eventually I yanked the power to the Sonoff dongle on my box where I had run scanner.py at one point, and the scanning went away. I confirmed by adding the following to `random_addr()` that the bdaddr sending the SCAN_REQs was the bdaddr printed out by scanner.py when I start it.

``` diff
diff --git a/python_cli/sniffle/sniffle_hw.py b/python_cli/sniffle/sniffle_hw.py
index 102b669..c738e81 100644
--- a/python_cli/sniffle/sniffle_hw.py
+++ b/python_cli/sniffle/sniffle_hw.py
@@ -441,6 +441,7 @@ class SniffleHW:
         addr[5] |= 0xC0 # make it static
         addr = bytes(addr)
         self.cmd_setaddr(addr)
+        print(f"SNIFFLE SET RANDOM_ADDR = {addr}")
         return addr
```

```
sudo python3 scanner.py -s /dev/ttyUSB0 -d
SNIFFLE SET RANDOM_ADDR = b'S\xca\xf5i\x82\xed'
Starting scanner. Press CTRL-C to stop scanning and show results.
Found 7F:67:9F:34:1B:E2 (RPA)...
<cut>
Found 11:22:33:44:55:66 (Public)...
^C

Scan Results:
================================================================================
<cut, but it shutdown gracefully on the latest checked in to master (whereas 1.10.0 shuts down with an error that I assume has been fixed)>
```

```
Timestamp: 3.765020  Length: 14  RSSI: -48  Channel: 37  PHY: 1M  CRC: 0x1894F7
Ad Type: SCAN_REQ
ChSel: 0 TxAdd: 1 RxAdd: 0 Ad Length: 12
ScanA: ED:82:69:F5:CA:53 (Static) AdvA: 11:22:33:44:55:66 (Public)
0x0000:  43 0c 53 ca f5 69 82 ed  66 55 44 33 22 11        C.S..i...<.o..
```

**Comment by sultanqasim** [comment]
This is a known/expected behaviour, though you're right, it would be better to stop firmware's scanning when stopping the scanner script.

**Comment by sultanqasim** [comment]
Should be fixed with 84b1ec0
